src/view/configuration/configuration.jsx

```diff
@@ -518,7 +520,22 @@ const Configuration = ({ formikProps, initInfo }) => {
                         />
                       </div>
                     </div>
-
+                    <div className="u-gapTop">
+                      <InfoTipLayout tip="Prehiding snipped should be placed on the page at the very top">
+                        <FieldLabel
+                          labelFor="prehidingStyleField"
+                          label="Default prehiding snippet (optional)"
+                        />
+                      </InfoTipLayout>
+                      <div className="u-gapLeft">
+                        <CopyToClipboard text={prehidingSnippet}>
+                          <Button
+                            data-test-id="copyToClipboardPrehidingStyleBtn"
+                            label="Copy now"
+                          />
```

Comment:
Should it say `Copy to Clipboard` ?

Reply:
I can rename it to `Copy to Clipboard`.

Reply:
Thank you @Aaronius for bringing this up, but there are other buttons that have the same look and feel, for example: the `Add Instance` button from the top right of the same view. 
This is because ` Open Editor` is a component which inherits `spectrum-Button--primary` CSS style:  
``` 
<Button
        {...otherProps}
        icon={<Code />}
        className={className}
        onClick={this.onClick}
        variant={invalid ? "warning" : "primary"}
      >
        Open Editor
      </Button>
```
If we want all the buttons from our extension to look the same we should add `variant: primary` attribute to all the buttons.
What do you think?

Reply:
Hmm, the editor button probably doesn't actually qualify as a "primary" button, so I would recommend we remove the "primary" variant from it.

Reply:
all `editorButtons` should be either "warning" or "secondary"?

Reply:
Probably, if that makes them consistent with other button like Copy to Clipboard. Optimally, we would have someone from UX provide these answers, so I'm just shooting in the dark.